### PR TITLE
376 global styling

### PR DESCRIPTION
**Pull request by Nikomus**
Trello card: https://trello.com/c/9iTymKQV/376-vvv-ee-global-styling

## Description

- Partial fix of the above
- Colours, fonts, some layout changes

@neontribe/sea-map

### CODE REVIEW

**Review by katjam — DISMISSED** [review]
Looks good. I am unconvinced about the `globalstyles` being decentralised, but not a problem.

Just checking also, the containers in theme and the images are not in use yet, right?

File: src/Page/NotAlone.elm [hunk 1]
```diff
@@ -108,4 +108,6 @@ journeyContentFromCardPosition cardPosition =
             , relatable = Journey6Relatable
             , hopeful = Journey6Hopeful
             , statement = Journey6Statement
+            , name = Journey6Name
+            , age = Journey6Age
             }
```
Comment (Nikomus) [inline src/Page/NotAlone.elm]:
```suggestion
            , statement = Journey6Statement
            }
```

**Comment by Nikomus** [comment]
The images and containers are for later. We may want to put the global styles back at a higher level, but at the moment they're being included when we constrain the whole app, and I'm not sure every page will be the same for that.

**Review by katjam — APPROVED** [review]
Go for it!

**Pull request merged**